scripts/travis-screenshots

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+# Decrypt the deployment key
+openssl aes-256-cbc -K $encrypted_1645300b04d0_key -iv $encrypted_1645300b04d0_iv -in .travis.sskey.enc -out .travis.sskey -d
```

Comment:
These are artifacts of travis encryption mechanism. Travis allows you to have a file available in the build, you run `travis encrypt-file some-file` and it gives you a `some-file.enc` file that you can commit safely, as it is encrypted. Inside the build, it exposes some environment variables that you can use to decrypt the file.

What I'm decrypting here is the deployment ssh key from github, so that the commits are not associated to any particular user.